src/cascadia/TerminalControl/ControlCore.idl

```diff
@@ -153,6 +153,6 @@ namespace Microsoft.Terminal.Control
         event Windows.Foundation.TypedEventHandler<Object, FoundResultsArgs> FoundMatch;
         event Windows.Foundation.TypedEventHandler<Object, ShowWindowArgs> ShowWindowChanged;
         event Windows.Foundation.TypedEventHandler<Object, UpdateSelectionMarkersEventArgs> UpdateSelectionMarkers;
-
+        event Windows.Foundation.TypedEventHandler<Object, OpenHyperlinkEventArgs> OpenHyperlink;

```

Comment:
FWIW: I had my feelings about ControlCore/Interactivity when I got to this line. I was mostly fine before I saw this, then it made me think twice. This is a user-interaction-originated event.